schemas/documents/shared/soldOutProjects.js

```diff
@@ -0,0 +1,18 @@
+export default {
+  title: 'Sold out projects',
+  name: 'soldOutProjects',
+  type: 'document',
+  __experimental_actions: ['update', 'create', /*'delete', */ 'publish'],
```

Comment:
let's leave out `__experimental_actions` here as this is only relevant for app/website pages
these are just regular documents, we can allow deletion in case of human error 
```suggestion
```